contribs/drt/src/main/java/org/matsim/contrib/drt/analysis/DrtAnalysisControlerListener.java

```diff
@@ -214,6 +205,38 @@ public void notifyIterationEnds(IterationEndsEvent event) {
 				filename(event, "drt_boardings", ".csv"), filename(event, "drt_alightments", ".csv"), network);
 	}
 
+	private String getMinCountIdleVehiclesOverDayFormatted() {
+		double minCountIdleVehiclesOverDay = Double.MAX_VALUE;
+		double[] stayTaskProfile = vehicleOccupancyProfileCalculator.getNonPassengerServingTaskProfiles().get(new DrtTaskType(DrtTaskBaseType.STAY));
+		double[] numberOfVehiclesInServiceProfile = vehicleOccupancyProfileCalculator.getNumberOfVehiclesInServiceProfile();
+		if (stayTaskProfile != null) {
+			for (int i = 0; i < numberOfVehiclesInServiceProfile.length; i++) {
+				if (numberOfVehiclesInServiceProfile[i] > 0) {
+					// only consider time intervals in which vehicles were in operation. Otherwise, any time period without a vehicle in operation will make for count 0.
+					minCountIdleVehiclesOverDay = Math.min(minCountIdleVehiclesOverDay, stayTaskProfile[i]);
+				}
+			}
+		}
+		return minCountIdleVehiclesOverDay < Double.MAX_VALUE ?
+				format.format(minCountIdleVehiclesOverDay) : notAvailableString;
+	}
+
+	private String getMinShareIdleVehiclesOverDayFormatted() {
+		double minShareIdleVehiclesOverDay = Double.MAX_VALUE;
+		double[] stayTaskProfile = vehicleOccupancyProfileCalculator.getNonPassengerServingTaskProfiles().get(new DrtTaskType(DrtTaskBaseType.STAY));
+		double[] numberOfVehiclesInServiceProfile = vehicleOccupancyProfileCalculator.getNumberOfVehiclesInServiceProfile();
```

Comment:
`numberOfVehiclesInServiceProfile` gets computed twice, i.e. in `getMinCountIdleVehiclesOverDayFormatted()` and `getMinShareIdleVehiclesOverDayFormatted()`. Maybe we want to compute it once and then pass to both methods. Quite likely, it's not a lot of computation, so we can keep it the way it is.

Reply:
Should be fixed by merging the two methods using numberOfVehiclesInServiceProfile to one method only.